buildscripts/include/path.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Get the directory of the current script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. "$SCRIPT_DIR/depinfo.sh"
```

Comment:
this is a bit redundant, you can just use `$DIR/include/depinfo.sh`

Reply:
Thanks for the review.
I opted for `$SCRIPT_DIR/depinfo.sh` over `$DIR/include/depinfo.sh `to ensure the script's portability and reliability.

 Using `$SCRIPT_DIR` guarantees that the script works regardless of its location, enhancing readability by explicitly stating the dependency's location relative to the script. 

Would you like me to continue with this approach, or do you have a preference for using `$DIR`?

Reply:
It's not like this script is designed to be moved, so yes I prefer relying on `$DIR`.

Reply:
Thank you i get it , I will update it now.